### DISCUSSION TITLE
Git, GitHub, Gitlab

### DISCUSSION THREAD

**Discussion by Ptr-Parker** [community/community#133935] [New to GitHub] [answered]
Can anyone tell me, or recommend me a way of learning the difference between Git, GitHub and GitLab, please?

**Comment by mrptrprkr** [comment, marked as answer]
Here is a link to a course, which explains the difference between Git, GitHub and GitLab: https://www.udemy.com/course/the-git-github-gitlab-bootcamp/?kw=git+github+gitlab&src=sac. I hope it helps

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @Ptr-Parker and @mrptrprkr. Just a heads up that, because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
